scalalib/src/JavaModule.scala

```diff
@@ -357,6 +357,11 @@ trait JavaModule
     })()
   }
 
+  /**
+   * Like [[resolvedIvyDeps]] but used only for compiling this module .
+   */
+  def resolvedCompilationIvyDeps: T[Agg[PathRef]] = resolvedIvyDeps
```

Comment:
I'm not that happy with this name. First, there is a high change of reading it as `resolveCompileIvyDeps` which would be the dual of `compileIvyDeps` (which currently does not exists). Also, the scaladoc gives no clear idea, why this is needed exactly and what's the difference to `resolvedIvyDeps`.